### ISSUE TITLE
Suggest change to URL format

### ISSUE THREAD

**Issue by RJWoodhead** [gabriel-dehan/dyson-sphere-blueprints#4]
There are several minor concerns about the URL format.

1) They are very long, and that length doesn't have any useful purpose.
2) If the title of a BP is changed, the URL no longer corresponds to the title.
3) Giving users control over the URLs can result in unexpected consequences, ie: dysonsphereblueprints.com/blueprints/are-for-losers. Or worse... much worse... :)

I would suggest a change to something simpler, like dysonsphereblueprints.com/blueprints/<username>/<sequence#>

**Comment by gabriel-dehan** [comment]
The slug indeed doesn't update when you update the title of a BP which can lead to some very weird things. The idea behind the slug was that it'd be better for SEO. 
In the end it's not a big deal nor is it an issue atm but I am considering it 👍

**Comment by RJWoodhead** [comment]
Understood. With respect to SEO concerns, the last time I checked, words in URLs are not a significant input to search engine algorithms; what counts is the title and actual visible page content. I took a quick look at the page source and it all looks cromulent from an SEO perspective (as much as that is useful for a specific site like this, you're always going to come up on page one for "dyson sphere blueprint"). The only thing I might suggest is adding the blueprint tags to the keywords meta tag, but that's sprinkles on an already iced cupcake.
